#
# This patch provides for possibility to build with zlib 1.3.
#
# Patch source: upstream
#
# Related issue is tracked by upstream bug
# https://bugzilla.mindrot.org/show_bug.cgi?id=3604
# which is to be fixed in future OpenSSH release.
# This patch can be removed once we upgrade to the fixed release.
#
--- old/configure.ac	2023-10-03 13:32:23.298008508 +0200
+++ new/configure.ac	2023-10-03 13:32:51.601296516 +0200
@@ -1488,7 +1488,7 @@ else
 	[[
 	int a=0, b=0, c=0, d=0, n, v;
 	n = sscanf(ZLIB_VERSION, "%d.%d.%d.%d", &a, &b, &c, &d);
-	if (n != 3 && n != 4)
+	if (n < 1)
 		exit(1);
 	v = a*1000000 + b*10000 + c*100 + d;
 	fprintf(stderr, "found zlib version %s (%d)\n", ZLIB_VERSION, v);
